controller_manager/test/test_controller_manager.cpp

```diff
@@ -253,5 +253,109 @@ TEST_P(TestControllerManager, per_controller_update_rate)
   EXPECT_EQ(test_controller->get_update_rate(), 4u);
 }
 
+TEST_P(TestControllerManager, per_controller_equal_and_higher_update_rate)
+{
+  auto strictness = GetParam().strictness;
+  auto test_controller = std::make_shared<test_controller::TestController>();
+
+  auto last_internal_counter = 0u;
+  for (unsigned int ctrl_update_rate : {100, 232, 400})
```

Comment:
I'll do a small refactor here tomorrow, otherwise we are good to go IMO

Reply:
Sure